### ISSUE TITLE
invenio_search not found during installation

### ISSUE THREAD

**Issue by wakudrle** [inveniosoftware/invenio-app-rdm#275]
Package version: v3.4.0a7

## Describe the bug
invenio_search not being installed from scripts. 

## Steps to Reproduce
Installing invenio using instructions at https://invenio-app-rdm.readthedocs.io/en/latest/installation.html. When I get to the step of ./scripts/botstrap. it install the invenio-app-rdm, but then gives the following dump. 

raceback (most recent call last):
  File "/home/bkudrle/.local/share/virtualenvs/invenio-app-rdm-v2tm59bf/bin/invenio", line 8, in <module>
    sys.exit(cli())
  File "/home/bkudrle/.local/share/virtualenvs/invenio-app-rdm-v2tm59bf/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/bkudrle/.local/share/virtualenvs/invenio-app-rdm-v2tm59bf/lib/python3.7/site-packages/flask/cli.py", line 586, in main
    return super(FlaskGroup, self).main(*args, **kwargs)
  File "/home/bkudrle/.local/share/virtualenvs/invenio-app-rdm-v2tm59bf/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/bkudrle/.local/share/virtualenvs/invenio-app-rdm-v2tm59bf/lib/python3.7/site-packages/click/core.py", line 1254, in invoke
    cmd_name, cmd, args = self.resolve_command(ctx, args)
  File "/home/bkudrle/.local/share/virtualenvs/invenio-app-rdm-v2tm59bf/lib/python3.7/site-packages/click/core.py", line 1297, in resolve_command
    cmd = self.get_command(ctx, cmd_name)
  File "/home/bkudrle/.local/share/virtualenvs/invenio-app-rdm-v2tm59bf/lib/python3.7/site-packages/flask/cli.py", line 527, in get_command
    self._load_plugin_commands()
  File "/home/bkudrle/.local/share/virtualenvs/invenio-app-rdm-v2tm59bf/lib/python3.7/site-packages/flask/cli.py", line 523, in _load_plugin_commands
    self.add_command(ep.load(), ep.name)
  File "/home/bkudrle/.local/share/virtualenvs/invenio-app-rdm-v2tm59bf/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2472, in load
    return self.resolve()
  File "/home/bkudrle/.local/share/virtualenvs/invenio-app-rdm-v2tm59bf/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2478, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/bkudrle/.local/share/virtualenvs/invenio-app-rdm-v2tm59bf/lib/python3.7/site-packages/invenio_rdm_records/__init__.py", line 11, in <module>
    from .ext import InvenioRDMRecords
  File "/home/bkudrle/.local/share/virtualenvs/invenio-app-rdm-v2tm59bf/lib/python3.7/site-packages/invenio_rdm_records/ext.py", line 13, in <module>
    from invenio_indexer.signals import before_record_index
  File "/home/bkudrle/.local/share/virtualenvs/invenio-app-rdm-v2tm59bf/lib/python3.7/site-packages/invenio_indexer/__init__.py", line 183, in <module>
    from .ext import InvenioIndexer
  File "/home/bkudrle/.local/share/virtualenvs/invenio-app-rdm-v2tm59bf/lib/python3.7/site-packages/invenio_indexer/ext.py", line 19, in <module>
    from .cli import run  # noqa
  File "/home/bkudrle/.local/share/virtualenvs/invenio-app-rdm-v2tm59bf/lib/python3.7/site-packages/invenio_indexer/cli.py", line 18, in <module>
    from invenio_search.cli import index
ModuleNotFoundError: No module named 'invenio_search'

## Expected behavior
Cannot run any step after that without getting same error.  

## Additional context
Ubuntu 18 desktop. Not very familiar with Python, so any hints are appreciated.

**Comment by fenekku** [comment]
Hi @wakudrle !

Sorry for the delay (I adjusted my notifications for faster response time in the future). Unfortunately the site you stumbled on should not be used at all. I just made an issue about removing it, because clearly it causes confusion and adds maintenance effort that we can't spare right now.

What you want to do instead is follow these much nicer docs: https://inveniordm.docs.cern.ch/ :smile: 

Let us know if you encounter problems with the setup described there.

Thanks for trying InvenioRDM out!
